tests/input_flows.py

```diff
@@ -291,6 +291,23 @@ def input_flow_t3t1(self) -> BRGeneratorType:
         self.debug.press_yes()
 
 
+class InputFlowSetBrightness(InputFlowBase):
+    def __init__(self, client: Client):
+        super().__init__(client)
+
+    def input_flow_tt(self) -> BRGeneratorType:
+        self.debug.press_yes()
+        yield
```

Comment:
...and if it wouldn't,  please assert the buttonrequest names everywhere:
```suggestion
        assert (yield).name == "whatever_is_appropriate"
```